src/de/grobox/liberario/fragments/DirectionsFragment.java

```diff
@@ -533,7 +564,7 @@ private void showMore(boolean animate) {
 
 	private void showLess(boolean animate) {
 		showingMore = false;
-		ui.via.setLocation(null);
+		ui.via.setLocation((Location)null);
```

Comment:
No this doesn't make it better, sorry :P

Maybe if you tell me what the problem is, I can help? Is it just a warning? In any case, if you want to pass `null` as `Location`, you should annotate the parameter of `setLocation()` method as `@Nullable`.